sumologic/resource_sumologic_token_test.go

```diff
@@ -180,11 +180,13 @@ resource "sumologic_token" "test" {
 func testAccCheckTokenAttributes(name string) resource.TestCheckFunc {
 	return func(s *terraform.State) error {
 		f := resource.ComposeTestCheckFunc(
+			resource.TestCheckResourceAttrSet(name, "id"),
 			resource.TestCheckResourceAttrSet(name, "name"),
 			resource.TestCheckResourceAttrSet(name, "description"),
 			resource.TestCheckResourceAttrSet(name, "status"),
 			resource.TestCheckResourceAttrSet(name, "type"),
 			resource.TestCheckResourceAttrSet(name, "version"),
+			resource.TestCheckResourceAttrSet(name, "encoded_token_and_url"),
```

Comment:
This method is called for both create and update. There we added the check for `encoded_token_and_url`.
I'm also testing this locally

Reply:
great thanks